sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/request/InitiateRefundRequest.java

```diff
@@ -0,0 +1,32 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.request;
+
+import org.json.JSONObject;
+
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.JsonConvertible;
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
+
+/**
+ * class for initiating refund
+ */
+public class InitiateRefundRequest  implements JsonConvertible {
+
+    private final Money money;
+    private final String description;
+    private final String vatCategory;
```

Comment:
The Online refund component will validate the vatCategory to be one of: HIGH, LOW or ZERO. Maybe it's best to create an enum for this field here as well. Although adding a new vatCategory type will require a change to the SDK. What do you think?